src/server.cpp

```diff
@@ -400,12 +400,8 @@ CServer::CServer ( const int          iNewMaxNumChan,
             QString().number( static_cast<int> ( iPortNumber ) ) );
     }
 
-    // enable jam recording (if requested) - kicks off the thread
-    if ( !strRecordingDirName.isEmpty() )
-    {
-        bRecorderInitialised = JamRecorder.Init ( this, iServerFrameSizeSamples );
-        bEnableRecording     = bRecorderInitialised;
-    }
+    // Enable jam recording (if requested) - kicks off the thread
```

Comment:
Why is "SetRecordingDir" enabling the recording? This is a bit confusing. Maybe rename the function.

Reply:
Ah yes, history...  The comment is now out of date.  If I'm pushing all of this code from the server.cpp into a new class in the recorder, then we'll be back to simply passing the directory through "quietly" when the server class instantiates.